_posts/en/newsletters/2019-07-31-newsletter.md

```diff
@@ -0,0 +1,211 @@
+      support in the future given that the way most wallets use bloom
+      filters creates a severe privacy leak.
+
+    - *DNS seeds can return only nodes signaling BIP111:* most P2P light
+      wallets query one or more Bitcoin DNS seeds for a list of nodes to
+      use.  Some seeds allow filtering which nodes they return by what
+      service bits that node has configured[^dns-query], with the
+      service bit for bloom filter support being specified by
+      [BIP111][].
+
+    - *BIP157 would use more bandwidth than BIP37:* it was suggested
+      that some users of [BIP37][] bloom filters will be unable to
+      switch to [BIP157][] compact block filters because of the later's
+      higher bandwidth usage.  Optech briefly investigated filter size
+      in [Newsletter #43][] and found the filters themselves to use less
+      than 100 MB a month.  Clients would also need to download matching
+      blocks, which would vary depending on their activity (roughly one
+      block for every sent or received transaction for typical users; most
+      blocks are less than 2 MB in size).  It's unclear how many users
+      would be significantly affected by having to download that amount
+      of data.
+
+    - *Interested parties can run their own nodes:* the change to
+      Bitcoin Core only disables serving bloom filters by default.  It
+      doesn't remove the feature.  Authors of wallets depending on bloom
+      filter support can easily operate their own nodes, and they can
+      also try to persuade other node operators who aren't worried about
+      the DoS vulnerability to set the configuration option to true,
+      e.g.  `peerbloomfilters=1`.
+
+## Bech32 sending support
+
+*Week 20 of 24 in a [series][bech32 series] about allowing the people
+you pay to access all of segwit's benefits.*
+
+{% include specials/bech32/20-percentage-loss.md %}
+
+## Selected Q&A from Bitcoin StackExchange
+
+*[Bitcoin StackExchange][bitcoin.se] is one of the first places Optech
+contributors look for answers to their questions---or when we have a
+few spare moments of time to help curious or confused users.  In
+this monthly feature, we highlight some of the top voted questions and
+answers made since our last update.*
+
+{% comment %}<!-- https://bitcoin.stackexchange.com/search?tab=votes&q=created%3a1m..%20is%3aanswer -->{% endcomment %}
+{% assign bse = "https://bitcoin.stackexchange.com/a/" %}
+
+FIXME
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[LND][lnd repo], [C-Lightning][c-lightning repo], [Eclair][eclair repo],
+[libsecp256k1][libsecp256k1 repo], [Bitcoin Improvement Proposals
+(BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [BIPs #800][] updates [BIP174][] to note that wallets must only sign
+  with the signature hash (sighash) type requested in a Partially-Signed
+  Bitcoin Transaction (PSBT) if the wallet consider that sighash type to
```

Comment:
- s/consider/considers/ ? (or is this some esoteric use of the subjunctive mood that I'm unfamiliar with?)
- I think this is unclear. It sounds like if the wallet considers the sighash to be unacceptable, it should default to signing with `SIGHASH_ALL`. That's not true. If it considers the sighash to be unacceptable, it should fail signing.